### PR TITLE
Upload validations

### PR DESCRIPTION
**Pull request by cdoyle-temple**
This will prevent networking errors from S3 such as: 

- https://app.honeybadger.io/projects/61882/faults/79111661

and others:

- https://app.honeybadger.io/projects/61882/faults/79055839
- https://app.honeybadger.io/projects/61882/faults/78896432
- https://app.honeybadger.io/projects/61882/faults/76725786
- https://app.honeybadger.io/projects/61882/faults/73583814

### CODE REVIEW

File: app/models/concerns/imageable.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@ module Imageable
 
   included do
     has_one_attached :image, dependent: :destroy
+    validates :image, content_type: { in: ["image/png", "image/jpg", "image/jpeg", "image/gif"],
+                                      message: I18n.t("manifold.error.content_type_invalid_image") },
+                      size: { less_than: 700.kilobyte ,
+                              message: I18n.t("manifold.error.file_size_out_of_range_image") }
```
Comment (nomadicoder) [inline app/models/concerns/imageable.rb:12]: - Tests if size exceeds 700kB
- Hard coded numbers (700) should be specified in `config`

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ def redirect_to_image(type)
       if @parent.image.attached?
         redirect_to url_for(@parent.send(type))
       else
-        raise ActionController::RoutingError.new("Image Not Found")
+        raise ActionController::RoutingError.new("Image Not Attached")
```
Comment (nomadicoder) [inline app/controllers/images_controller.rb:26]: Tests?

**Review by nomadicoder — CHANGES_REQUESTED** [review]
- Are we testing attachment errors, specifically, Not attached and size > 700kb?

File: app/controllers/images_controller.rb (hunk 2 above)
Comment (cdoyle-temple) [reply app/controllers/images_controller.rb:26]: Beyond this: spec/suport/imageable_spec.rb:27 what should we test for? The text?

File: app/models/concerns/imageable.rb (hunk 1 above)
Comment (cdoyle-temple) [reply app/models/concerns/imageable.rb:12]: Added test and numeric sizes to config.

**Review by nomadicoder — APPROVED** [review]

**Pull request merged**